src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -3791,9 +3900,24 @@ namespace ZoneTempPredictorCorrector {
 		static Real64 TempDepCoef( 0.0 ); // Formerly CoefSumha, coef in zone temp equation with dimensions of h*A
 		static Real64 TempIndCoef( 0.0 ); // Formerly CoefSumhat, coef in zone temp equation with dimensions of h*A(T1
 		static Real64 AirCap( 0.0 ); // Formerly CoefAirrat, coef in zone temp eqn with dim of "air power capacity"
+		static Real64 AirCapHM(0.0); // Air power capacity for hybrid modeling 
 		static Real64 SNLoad( 0.0 ); // Sensible load calculated for zone in watts and then loaded in report variables
 		static int ZoneNum( 0 );
 		static int ZoneNodeNum( 0 ); // System node number for air flow through zone either by system or as a plenum
+		
+		// Added for hybrid model by Sang Hoon Lee, July 2015
+		static int HMStartDay( 0 ); // Temporary date   
```

Comment:
Moved the hybrid modeling variables in the section where they are used. Also removed static.